### PR TITLE
Ansible 1.8.4

### PR DESCRIPTION
**Pull request by PikachuEXE**
Ansible 1.9.x is breaking for our deployment process somehow
And we don't have time to investigate yet

### CODE REVIEW

File: ansible18.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
+class Ansible18 < Formula
+  desc "The simplest way to automate apps and IT infrastructure. Version 1.8.x"
```
Comment (DomT4) [inline ansible18.rb]: This is too long. `"name: desc" should be less than 80 characters (currently 99).`

**Comment by DomT4** [comment]
Please add a `conflicts_with "ansible", :because => "Differing versions of same formula"` line one blank line after the last dependency.

**Comment by PikachuEXE** [comment]
I cannot get the build to pass

Always the same error message
```
Checking out Revision d9c7d915bd38ba7a4f4e191fc8ea366919fae224 (refs/remotes/origin/pr/896/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f d9c7d915bd38ba7a4f4e191fc8ea366919fae224
FATAL: Could not checkout null with start point d9c7d915bd38ba7a4f4e191fc8ea366919fae224
hudson.plugins.git.GitException: Could not checkout null with start point d9c7d915bd38ba7a4f4e191fc8ea366919fae224
```

**Comment by DomT4** [comment]
```
Error: 1 problem in 1 formula

==> audit problems
homebrew/versions/ansible18:
 * Description is too long. "name: desc" should be less than 80 characters (currently 99).
```

**Comment by PikachuEXE** [comment]
`The simplest way to automate apps and IT infrastructure. Version 1.8.x` has 99 chars!?

**Comment by DomT4** [comment]
Erm. No. It has 70. What is going on :eyes:

**Comment by DomT4** [comment]
Oh, I see how we're counting this. We work out the length by adding the full formula name, in this case `homebrew/versions/ansible18` to the length of the `desc`. We could probably be clearer about that.

**Comment by PikachuEXE** [comment]
Just changed description to remove all audit issues.
But still failed

**Comment by DomT4** [comment]
Test-bot is going through its moody, rebellious teenager phase. I've asked it to try the build again.

**Comment by PikachuEXE** [comment]
The build has passed now :)

**Comment by DomT4** [comment]
And now merged in https://github.com/Homebrew/homebrew-versions/commit/6d2beec734cf127995863b80acc157510d337340 :wink:. Thank you for your contribution to Versions, we appreciate it! :smiley_cat:.

It's a nit, but commit messages for new formula should be styled `ansible18 1.8.4 (new formula)` or similar, Thanks!

**Comment by PikachuEXE** [comment]
Please take a look at #899 to also let other potential contributors to know about this :wink:

**Pull request closed without merging**